frontend/src/metabase/databases/components/DatabaseForm/DatabaseForm.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useLayoutEffect, useMemo, useState } from "react";
+import { t } from "ttag";
+import Form from "metabase/core/components/Form";
+import FormProvider from "metabase/core/components/FormProvider";
+import FormSubmitButton from "metabase/core/components/FormSubmitButton";
+import FormErrorMessage from "metabase/core/components/FormErrorMessage";
+import { DatabaseData, Engine } from "metabase-types/api";
+import DatabaseEngineField from "../DatabaseEngineField";
+import DatabaseNameField from "../DatabaseNameField";
+import DatabaseDetailField from "../DatabaseDetailField";
+import { getSchema } from "../../utils/schema";
+import { getVisibleFields } from "../../utils/visibility";
```

Comment:
Should we follow the rule that the closer the import is to the source file, the lower it is in the imports list?
If we do, `utils` should go before `Database*Field` imports

Reply:
Fixed